app/src/app/content-plugin/services/content-management.service.ts

```diff
@@ -1083,4 +1088,10 @@ export class ContentManagementService {
   private static focusCreateMenuButton(): void {
     document.querySelector<HTMLElement>('app-create-menu button').focus();
   }
+
+  private static focusAfterClose(focusedElementSelector: string): void {
```

Comment:
why does it have to be `static` if it's private, you can remove the `static` and reduce the usage of `ContentManagementService.focusAfterClose` to just a method call.

Reply:
done